utils/hare-shutdown

```diff
@@ -95,9 +95,18 @@ def processes_by_consul_svc_name(cns: Consul) -> Dict[str, List[Process]]:
             from e
 
 
+def get_node_name() -> str:
```

Comment:
Similar function is added to utils/utils.py by Shreya. Kindly rebase.

Reply:
There is no such a file in `cortx-1.0`. I guess Shreya's refactoring was not merged to `cortx-1.0`,

Reply:
Ah, yes sorry, its not in cortx-1.0. Alright.